### ISSUE TITLE
Conflicting images

### ISSUE THREAD

**Issue by kotarohara** [ProjectSidewalk/SidewalkWebpage#10]
Jon's feedback: There are conflicts between multiple images that capture same location; one image has accessibility problem but another don't (e.g., problem like lack of curb ramps being fixed). What should users do for this.

**Comment by kotarohara** [comment]
They should label whatever they see in the Street View image. Not sure if this is really a problem, so I'll close the issue for now.
